airbyte-integrations/connectors/source-linkedin-pages/metadata.yaml

```
@@ -41,6 +41,4 @@ data:
           secretStore:
             type: GSM
             alias: airbyte-connector-testing-secret-store
-  connectorBuildOptions:
-    baseImage: docker.io/airbyte/python-connector-base:1.2.2@sha256:57703de3b4c4204bd68a7b13c9300f8e03c0189bffddaffc796f1da25d2dbea0
-metadataSpecVersion: "1.0"
+  metadataSpecVersion: "1.0"
```

Looks like a bug, I think it should be top level
```suggestion
metadataSpecVersion: "1.0"
```

---

Right, my bad